grapher/core/GrapherAnalytics.ts

```diff
@@ -1,4 +1,4 @@
-const DEBUG = false
+const DEBUG = true
```

Comment:
left over?

Reply:
good catch, thanks!

Reply:
by the way, I think this should probably be an env settings to avoid these situations.

Reply:
Haha I've done it to a few times. The problem with adding an env setting is it makes what otherwise is something dead simple more complex. I wonder if there's an eslint rule we could add, like:
```
// eslint-next-line-cannot-contain true
const DEBUG = false
``
So that you'd get a fail if you checked it in.

Reply:
:) I see what you mean. Yep that's something worth looking into.